docs/user/interactive-running-locally.rst

```diff
@@ -0,0 +1,121 @@
+
+.. note::
+
+    All of the instructions below assume that you have cloned your project
+    locally and that you have `Docker
+    installed <https://docs.docker.com/get-docker/>`_. On Linux systems, Docker
+    requires root privileges - if you do not have root access to the host, you
+    might consider `rootless
+    Docker <https://docs.docker.com/engine/security/rootless/>`_.
+
+
+Building your own image
+-----------------------
+
+If you are working on a project that has not yet been pushed to the server,
+or you just prefer to build the image locally, you can certainly do so
+with Docker. To build the image, run this command from the root directory
+of your project:
+
+.. code-block::
+
+    $ docker build -t <imageName>:<tag> .
+
+The ``imageName`` and ``tag`` can be anything you want, but you might want to
+try to use something you will remember. To see which images you have in
+your local Docker registry, run
+
+.. code-block::
+
+    $ docker images
+
+
+Using the images built on RenkuLab
+----------------------------------
+
+If your repository on RenkuLab is public, the easiest way to spin up your
+runtime environment locally is by pulling the image that has already been built.
+First, make sure that your project has been pushed and that the image build is
+complete. If you are not sure, the easiest way to check is to navigate to your
+project in RenkuLab and go to launch a new session - if the "Start environment"
+button is available, it means the image has been built already.
+
+To pull the image, you need to construct the image name. This is typically the
+format to expect: ``registry.<renku-host>/<namespace>/<project>:<commit-sha>``.
+Lets break down the meaning of these terms in brackets:
+
+* ``renku-host`` is the hostname of the RenkuLab instance, e.g. ``renkulab.io``
+* ``namespace`` is either the username or the group name of the project owner
+* ``project`` is the project name
+* ``commit-sha`` is the first 7 characters of the ``commit-sha``
+
+``namespace`` and ``project`` are easy - if your repository remote is something
+like ``https://renkulab.io/gitlab/rok.roskar/flights-tutorial.git`` then the
+``<namespace>/<project>`` is ``rok.roskar/flights-tutorial``.
+
+You can get the ``commit-sha`` by running ``git log`` in your repository, e.g.
+
+.. code-block::
+
+    $ git log
+
+    commit 53cbd6a11dc5b0d139b2563b5aad83ae4f676b18 (HEAD -> master, origin/master, origin/HEAD)
+    Author: Rok Roskar <rok.roskar@sdsc.ethz.ch>
+    Date:   Tue Nov 3 17:15:31 2020 +0100
+
+    renku graph generate
+
+The ``commit-sha`` here is ``53cbd6a11dc5b0d139b2563b5aad83ae4f676b18`` so the full
+name of this image, including the tag would be
+``registry.renkulab.io/rok.roskar/flights-tutorial:53cbd6a``.
+
+
+Launching the interactive environment locally
+---------------------------------------------
+
+Finally, if we wanted to launch an environment locally with the image that was
+built for this commit we can do
```

Comment:
It's probably obvious, but I would specify that the command should be executed from the cloned project main folder